oauthdoctor/oauth/helper_test.go

```diff
@@ -0,0 +1,42 @@
+package oauth
+
+import (
+	"io/ioutil"
+	"log"
+	"testing"
+)
+
+func TestReadCustomerID(t *testing.T) {
+	log.SetOutput(ioutil.Discard)
+
+	original := readStdin
+	defer func() { readStdin = original }()
+
+	tests := []struct {
+		desc  string
+		input string
+		want  string
+	}{
+		{
+			desc:  "Return a valid customer ID",
+			input: "123-456-7890",
+			want:  "1234567890",
+		},
+		{
+			desc:  "Return original string",
+			input: "abc",
+			want:  "abc",
+		},
+	}
+
+	for _, test := range tests {
+		readStdin = func() string {
+			return test.input
+		}
+
+		got := ReadCustomerID()
+		if got != test.want {
+			t.Errorf("[%s] want: %s, got: %s\n", test.desc, test.want, got)
```

Comment:
The order should be got then want.
https://g3doc.corp.google.com/codelab/unittesting/go/g3doc/index.md?cl=head